.github/workflows/tests.yml

```diff
@@ -76,3 +76,18 @@ jobs:
         run: mv chore/travis/logback.xml src/test/resources/
       - name: Run extra checks
         run: ./chore/travis/travis-extra.sh
+
+  depclean:
+    runs-on: ubuntu-latest
+    env:
+      MAVEN_OPTS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false
+    name: Dependency usage analysis
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+      - uses: actions/setup-java@d202f5dbf7256730fb690ec59f6381650114feb2 # v1.4.3
+        with:
+          java-version: 11
+      - name: Use silent log config
+        run: mv chore/travis/logback.xml src/test/resources/
+      - name: Build and test
+        run: mvn package -DskipTests -Pdepclean
```

Comment:
This still causes duplication. It's rather confusing, but `travis-extra.sh` is actually executed in the GitHub Actions `extra` job, so just having `mvn package -DskipTests -Pdepclean` in there is sufficient to run it with our current CI setup.

Sorry about the confusing setup, it may or may not be my fault :). I'm gonna fix it ... #3836

Reply:
Thank @slarse. This is indeed confusing. 
To make it more clear, I've renamed the references to the travis scripts, from `travis-*` to `ci-*`.
And of course, I removed the redundant depclean GH actions job (now added to `ci-extra.sh`)